tests/filters/parser_filter.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Tests for the helper for parser and plugin filter expressions."""
+
+from __future__ import unicode_literals
+
+import unittest
+
+from plaso.filters import parser_filter
+from plaso.parsers import presets as parsers_presets
+
+from tests.filters import test_lib
+
+
+class ParserFilterExpressionHelperTest(test_lib.FilterTestCase):
+  """Tests for the helper for parser and plugin filter expressions."""
+
+  # pylint: disable=protected-access
+
+  def testGetParserAndPluginsList(self):
+    """Tests the _GetParserAndPluginsList function."""
+    presets_manager = parsers_presets.ParserPresetsManager()
+
+    test_helper = parser_filter.ParserFilterExpressionHelper(presets_manager)
+
+    expression = test_helper._GetParserAndPluginsList(
+        {'excluded': set(['*', 'plugin1'])})
+    self.assertEqual(expression, ['excluded', 'excluded/plugin1'])
+
+  def testJoinExpression(self):
+    """Tests the _JoinExpression function."""
+    presets_manager = parsers_presets.ParserPresetsManager()
+
+    test_helper = parser_filter.ParserFilterExpressionHelper(presets_manager)
+
+    expression = test_helper._JoinExpression({'excluded': set(['*'])}, {})
```

Comment:
It would be a lot easier to read what these asserts are meant to test if every test expression reflect what was trying to be tested, rather than just being called "expression". It's not necessary to change all these, but please keep it mind for future test methods where a large number of things are being tested for, as is the case here.

Reply:
ack